drivers/sensor/vl53l0x/vl53l0x_i2c_platform.h

```diff
@@ -0,0 +1,378 @@
+/* vl53l0x_i2c_platform.h - Zephyr customization of ST vl53l0x library,
+ * i2c functions prototypes
+ */
+
+/*
+ * Copyright (c) 2017 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _VL53L0X_I2C_PLATFORM_H_
+#define _VL53L0X_I2C_PLATFORM_H_
+
+#include "vl53l0x_def.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
+#include <stdarg.h>
+
+#define	   I2C                0x01
```

Comment:
How much of anything in this file is used?  It seems as if none of it is - if so can we remove it?